### PR TITLE
Spelling docs

### PR DESCRIPTION
**Pull request by jsoref**
This fixes some misspellings in Swift documentation, it is split per https://github.com/apple/swift/pull/42421#issuecomment-1101092698

### CODE REVIEW

**Comment by jsoref** [comment]
I think this is supposed to be split between user facing documentation and compiler documentation, but I'm having trouble figuring out what that is. When someone identifies which files are which, I'll split this PR.

File: docs/refactoring/SwiftLocalRefactoring.md [hunk 1]
```diff
@@ -300,7 +300,7 @@ with (2). It passes if the two are identical, otherwise the test fails.
 3  }
 4  // RUN: rm -rf %t.result && mkdir -p %t.result
 5  // RUN: %refactor -localize-string -source-filename %s -pos=2:14 > %t.result/localized.swift
-6  // RUN: diff -u %S/Iutputs/localized.swift.expected %t.result/localized.swift
+6  // RUN: diff -u %S/Outputs/localized.swift.expected %t.result/localized.swift
```
Comment (CodaFi) [inline docs/refactoring/SwiftLocalRefactoring.md:303]: Neat - good catch

**Review by CodaFi — APPROVED** [review]

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by CodaFi** [comment]
Note to self: squash merge this

**Comment by CodaFi** [comment]
Windows failure is unrelated to these changes

⛵

**Pull request merged**